kolibri/core/assets/src/views/ContentRenderer/__tests__/utils.spec.js

```diff
@@ -0,0 +1,80 @@
+import Vue from 'vue';
+import { canRenderContent, getRenderableFiles, getDefaultFile, getFilePreset } from '../utils';
+import { RENDERER_SUFFIX } from '../constants';
+
+// Add a component to the Vue instance that can be used to test the utility functions
+const addRegisterableComponents = (...presets) => {
+  presets.forEach(preset => {
+    Vue.component(preset + RENDERER_SUFFIX, { template: '<div></div>' });
+  });
+};
+
+describe('Utility Functions', () => {
+  beforeEach(() => {
+    Vue.options.components = {};
+  });
+
+  describe('canRenderContent', () => {
+    test('returns true if preset renderer component is registered', () => {
+      addRegisterableComponents('preset1');
+      expect(canRenderContent('preset1')).toBe(true);
+    });
+
+    test('returns false if preset renderer component is not registered', () => {
+      expect(canRenderContent('preset2')).toBe(false);
+    });
+  });
+
+  describe('getRenderableFiles', () => {
+    test('returns renderable files', () => {
```

Comment:
... that are available, can be rendered and doesnt have a thumbnail. So we can know what are we expecting from the results.

Reply:
Got it! Thanks.